apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -125,12 +123,7 @@
         return getThreadsStream.cancel(reason);
       };
 
-      var draftsPromise = Drafts.request().then(() => {
```

Comment:
I tend to move this part to a separate bug, it turned out to be complex to land conversation service in master with new way of draft handling